Finance/currency-tracker.1h.py

```diff
@@ -1,38 +0,0 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-# <xbar.title>Currency Tracker</xbar.title>
-# <xbar.version>1.0</xbar.version>
-# <xbar.author>Maxime Bertheau</xbar.author>
-# <xbar.author.github>maxoumime</xbar.author.github>
-# <xbar.desc>Keep an eye on the currencies you choose from your menu bar !</xbar.desc>
-# <xbar.image>https://nothingreally.botler.me/bitbar.currency-tracker.png</xbar.image>
-
-import urllib2
-import json
-
-# Write here the currencies you want to see
-
-# Base comparaison currency
-currFrom = "EUR"
-# Array of tracked currencies
-currTo = ["CAD", "USD"]
-
-urlParamTo = currTo[0]
-if len(currTo) > 1:
-    urlParamTo = ",".join(currTo)
-
-url = "https://api.exchangeratesapi.io/latest?base=" + currFrom + "&symbols=" + urlParamTo
```

Comment:
This still seems to be a valid API. What is the reason behind removing this plugin?

Reply:
The API is no longer open and the free tier only allows for 250 requests per month.

Reply:
🤔 I'm not sure we want to remove this entirely. Free tier would still allow you to track things, albeit not on an hourly schedule.